src/MessagingAttachment/MessagingAttachment.less

```diff
@@ -11,6 +11,63 @@
   .text--truncate();
 }
 
+// Family portal specific styling
+.MessagingAttachment--ParentContainer.MessagingAttachment--FamilyPortal {
+  border-radius: @size_m;
+  background-color: @neutral_white;
+  box-shadow: 0px @size_3xs 6px rgba(71, 76, 94, 0.3);
+  margin-top: @size_2xs;
+
+  .MessagingAttachment--Container {
+    border: unset;
+    border-radius: unset;
+
+    .padding--y--s();
+    .padding--x--xs();
+  }
+
+  .MessagingAttachment--IconContainer {
+    margin-right: 0.5rem;
+  }
+
+  .MessagingAttachment--Title {
+    font-size: 0.875rem;
+  }
+}
+
+// Attachment-only messages
+.MessagingBubble--HideBubble
+  .MessagingAttachment--ParentContainer.MessagingAttachment--FamilyPortal:first-of-type {
+  margin-top: 0;
+}
+
+// Replying to an announcement that countains an attachment
+.QuotedAnnouncementBubble--container {
+  .MessagingAttachment--ParentContainer.MessagingAttachment--FamilyPortal:first-of-type {
```

Comment:
I see that this first classname in this group includes the theme, but all subsequent styles in this file aren't gated by theme. Is that intentional? (Just sanity checking)

Reply:
hmm which theme? `.MessagingAttachment--FamilyPortal`?

Reply:
Yup exactly! So the `margin-top: 0` on the next line will only get applied when the theme is `familyPortal` but all styles that come after will get applied everywhere, regardless of theme. Which again may be intentional! Just sanity checking that we're intentionally changing styles outside of FamPo. As we discussed in person, I'm all for applying style changes globally where possible, so this may be okay.

Reply:
If so, then yes. The theme class gets applied to the top level container of the component, then subsequent classes are updated based on that

Reply:
Unless I'm confused, which I maybe am

Reply:
I'm almost wondering the opposite, too. Should the `margin-top: 0` on the next line also get applied regardless of theme (removing the `.MessagingAttachment--FamilyPortal` above)?